deploy/examples/create-external-cluster-resources.py

```diff
@@ -1128,7 +1128,7 @@ def create_rgw_admin_ops_user(self):
                 )
                 raise Exception(err_msg)
 
-        # separately add info=read caps, because sometimes users already exited and the cap doesn't update
+        # separately add info=read caps for rgw-endpoint ip calidation
```

Comment:
@travisn see from this line 1131 to 1157 here is the check for ceph version and adding it if supports it